src/draw/lv_draw_dither.h

```diff
@@ -0,0 +1,71 @@
+/**
```

Comment:
I think the dithering related files should go into the `src/draw/sw` folder.

Reply:
It's not software related, is it ?
The dithering algorithm builds a pixel map that's blitted by the backend. So a GPU backend will just optimize this, IIUC.
The map is constructed by a software algorithm, but I had the impression that the sw folder was for sw rendering algorithms, not for higher level sw drawing functions. Please advise.

Reply:
IMO it's related to SW renderer. E.g. ThorVG would have it's own gradient functions.

Reply:
Ok, changed.